homeassistant/helpers/config_validation.py

```diff
@@ -219,6 +221,17 @@ def validate(obj):
     return validate
 
 
+def validate_values(valid_values):
```

Comment:
ah, try `[vol.In(SENSOR_TYPES.keys()]` and if that doesn't work `vol.All(vol.Coerce(list), [vol.In(SENSOR_TYPES.keys())])`. And once you find something that works it might be useful to try it without the `.keys()`, we are going to need this in several other places (nest sensors for one).